src/controllers/StoryController.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace App\Controllers;
+
+use App\Exceptions\StoryNotFoundException;
+use App\Models\Story;
+use App\Models\Comment;
+
+class StoryController
+{
+
+    protected $storyModel;
+    protected $commentModel;
+
+    public function __construct($config)
+    {
+        $this->storyModel   = new Story($config);
```

Comment:
Remember that object creation is a job, too. That means creating the objects here violates the SRP.